pkg/minikube/tunnel/tunnel.go

```diff
@@ -18,6 +18,12 @@ type LoadBalancerTunnelState struct {
 	ServicesWithLoadBalancer []core_v1.Service
 }
 
-func (tunnelState *LoadBalancerTunnelState) next() *LoadBalancerTunnelState {
-	return tunnelState
+type LoadBalancerTunnelStateUpdate struct {
```

Comment:
nit:  this name is a bit verbose, perhaps TunnelStateUpdate, LBTunnelStateUpdate, or LoadBalancerUpdate would make sense for these?

Reply:
sounds good, we can rename it!

Reply:
`TunnelStateUpdate`